extra-python/hstspreload/spec

```diff
@@ -0,0 +1,3 @@
+VER=20200717
+SRCTBL="https://github.com/sethmlarson/hstspreload/archive/2020.7.17.tar.gz"
```

Comment:
Hard-coded version in source link, please use `2020.7.17` as version number (why did you bother changing this?).

Reply:
789c19a